pep-0655.rst

```diff
@@ -388,16 +397,16 @@ module.
 Rejected Ideas
 ==============
 
-Special syntax around the *key* of a TypedDict item
----------------------------------------------------
+Special notation around the *key* of a TypedDict item
+-----------------------------------------------------
 
 ::
 
    class MyThing(TypedDict):
        opt1?: str  # may not exist, but if exists, value is string
        opt2: Optional[str]  # always exists, but may have None value
 
-This syntax would require Python grammar changes and it is not
+This notation would require Python grammar changes and it is not
```

Comment:
```suggestion
This notation would require Python grammar changes, and it is not
```

Missing comma

Reply:
Feels like this comma interrupts the flow unnecessarily. Ignoring.